### DISCUSSION TITLE
AKS dashboard maintenance

### DISCUSSION THREAD

**Discussion by d-ariho** [Azure/CCOInsights#295] [Q&A] [answered]
Hi Team,
Is there any chance of an update about the maintenance of the AKS dashboards. I understand there is an issue with expired APIs and most of the data is coming up as blank. Is this being worked on or not at all.

Thanks

**Comment by josunefon** [comment, marked as answer]
Hi @d-ariho ,
Sadly, this dashboard is discontinued, and it will not be evolved/maintained. 
We decided to keep it in the repo because the engine to retrieve data from log analytics it can be useful for future developments.
